.github/workflows/ci.yaml

```diff
@@ -20,13 +20,14 @@ jobs:
 
       - name: Installing packages Angular
         run: cd ${{ github.workspace }}/angular && npm install
-      - name: Installing packages React
-        run: cd ${{ github.workspace }}/react && npm install
 
       - name: Run test
         uses: GabrielBB/xvfb-action@v1
         with:
           run: cd ${{ github.workspace }}/angular && npm run test
+      - name: Test passing check

```

Comment:
reference step by "id", so it's readable

Reply:
See how to reference steps by ID and then check if it's failed or not
https://stackoverflow.com/questions/57850553/github-actions-check-steps-status